code/modules/jobs/job_types/_job.dm

```diff
@@ -128,7 +128,7 @@
 	if(!H)
 		return FALSE
 	
-	if(CONFIG_GET(flag/enforce_human_authority) && (title in GLOB.command_positions))
+	if(CONFIG_GET(flag/enforce_human_authority) && (title in GLOB.command_positions) && (title != "Blueshield"))
```

Comment:
has to be a better way of making this work this is kind of snowflake

Reply:
well, this is really the only position that would ever be "command" without human restrictions. if it ever comes that there is another position or if human authority is disabled, we can just remove this